doc/source/reference/arrays.datetime.rst

```diff
@@ -67,7 +67,8 @@ datetime type with generic units.
     array(['2007-07-13', '2006-01-13', '2010-08-13'], dtype='datetime64[D]')
 
     >>> np.array(['2001-01-01T12:00', '2002-02-03T13:56:03.172'], dtype='datetime64')
-    array(['2001-01-01T12:00:00.000-0600', '2002-02-03T13:56:03.172-0600'], dtype='datetime64[ms]')
+    array(['2001-01-01T12:00:00.000', '2002-02-03T13:56:03.172'],
+          dtype='datetime64[ms]')
```

Comment:
This is a bit of a weird one! I'm hoping the old behavior was a bug.

Reply:
The timezone part is from NumPy < 1.11.